README.md

```diff
@@ -1,5 +1,5 @@
 # `mips_to_c`
-Given some MIPS assembly, this program will attempt to convert it to C.
+Given some MIPS or PPC assembly, this program will attempt to convert it to C.
```

Comment:
@simonlindholm - I know you wanted me to update the README, do you think the changes I made are good enough, or are there areas I should expand on?

Reply:
I think it's good enough for now.